Description: Build PDF documentation with standard fonts.
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2013-10-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/docsrc/asn1c-usage.tex
+++ b/doc/docsrc/asn1c-usage.tex
@@ -6,9 +6,6 @@
 \usepackage[hmargin={1in,1in},vmargin={2in,2in}]{geometry}
 
 \defaultfontfeatures{Mapping=tex-text}
-\setmainfont{Myriad Pro}
-\setsansfont{Myriad Pro}
-\setmonofont{Consolas}
 
 \usepackage{fancyhdr}
 \pagestyle{fancy}
